modules/utxo-lib/src/bitgo/transaction.ts

```diff
@@ -32,23 +32,47 @@ export function createTransactionFromHex(hex: string, network: Network): UtxoTra
   return createTransactionFromBuffer(Buffer.from(hex, 'hex'), network);
 }
 
-export function setTransactionBuilderDefaults(txb: UtxoTransactionBuilder, network: Network): void {
+export function getDefaultTransactionVersion(network: Network): number {
   switch (getMainnet(network)) {
     case networks.bitcoincash:
     case networks.bitcoinsv:
     case networks.bitcoingold:
-      txb.setVersion(2);
+      return 2;
+    case networks.zcash:
+      return 4;
```

Comment:
we will change this to `return isTestnet(network) ? 5 : 4` in a follow-up

Reply:
make that `return isTestnet(network) ? 450 : 400` with  https://github.com/BitGo/BitGoJS/pull/1843